src/applications/vaos/new-appointment/newAppointmentFlow.js

```diff
@@ -292,3 +295,140 @@ export default {
     url: '/new-appointment/review',
   },
 };
+
+/**
+ * Function to get new appointment workflow.
+ * The URL displayed in the browser address bar is changed when the feature flag
+ * is true.
+ *
+ * @export
+ * @param {boolean} state - New appointment state
+ * @returns {object} Appointment workflow object
+ */
+export default function getNewAppointmentFlow(state) {
+  const featureBreadcrumbUrlUpdate = selectFeatureBreadcrumbUrlUpdate(state);
+
+  return {
+    ...flow,
+    appointmentTime: {
+      ...flow.appointmentTime,
+      url: featureBreadcrumbUrlUpdate
+        ? 'appointment-time'
+        : '/new-appointment/appointment-time',
+    },
+    audiologyCareType: {
+      ...flow.audiologyCareType,
+      url: featureBreadcrumbUrlUpdate
+        ? 'audiology-care'
+        : '/new-appointment/audiology',
+    },
+    ccClosestCity: {
+      ...flow.ccClosestCity,
+      url: featureBreadcrumbUrlUpdate
+        ? 'choose-closest-city'
+        : '/new-appointment/choose-closest-city',
+    },
```

Comment:
This is a dumb question. I see the old site goes to the url 
`/new-appointment/choose-closest-city` because it is coded above. But how does the new site know to go `/schedule/choose-closest-city` when "/schedule/" is excluded?

Reply:
Good question. I won't go to /schedule/choose-closest-city if the prefix is excluded. I think the ideal solution is to change the basename on the Router but this is not possible.